pkg/controller/logstorage/logstorage_controller.go

```diff
@@ -599,62 +599,53 @@ func (r *ReconcileLogStorage) getElasticsearchCertificateSecrets(ctx context.Con
 	return esKeyCert, certSecret, err
 }
 
-func (r *ReconcileLogStorage) kibanaInternalSecrets(ctx context.Context, instl *operatorv1.InstallationSpec) ([]*corev1.Secret, error) {
+func (r *ReconcileLogStorage) kibanaInternalSecrets(ctx context.Context, instl *operatorv1.InstallationSpec) (*corev1.Secret, *corev1.Secret, bool, error) {
```

Comment:
I think you should change the order of the return values to `*corev1.Secret, bool, *corev1.Secret, error` since the bool is telling us about the 1st secret.

Reply:
Also please add a comment explaining the values returned from this function.